app/test/qgb_rpc_test.go

```diff
@@ -19,7 +19,9 @@ func TestBlobstreamRPCQueries(t *testing.T) {
 		t.Skip("skipping blobstream integration test in short mode.")
 	}
 	ecfg := encoding.MakeConfig(app.ModuleEncodingRegisters...)
-	cfg := testnode.DefaultConfig().WithModifiers(genesis.SetDataCommitmentWindow(ecfg.Codec, 100))
+	cfg := testnode.DefaultConfig().
+		WithModifiers(genesis.SetDataCommitmentWindow(ecfg.Codec, 100)).
+		WithConsensusParams(app.DefaultInitialConsensusParams())
```

Comment:
why are we adding the consensus params here?

Reply:
We're adding the DefaultInitialConsensusParams here to start this test on app version 1. Without this modification, the test would start with DefaultConsensusParams which uses app version 2 where blobstream is disabled.

Ref: https://github.com/rootulp/celestia-app/blob/e7be31bb75eadec0cdc9a98a43cb645a92684e17/app/default_overrides.go#L228-L237